extensions/PhabBugz/lib/Feed.pm

```diff
@@ -358,6 +358,141 @@ sub _is_uplift_request_form_change {
   return $story_text =~ /\s+uplift request field/;
 }
 
+sub readable_answer {
+  my ($answer) = @_;
+
+  # Return the value itself if it is not a number.
+  if ($answer ne '0' && $answer ne '1') {
+    return $answer;
+  }
+
+  # Return 'yes' for `1`.
+  if ($answer) {
+    return 'yes';
+  }
+
+  # Return 'no' for `0`.
+  return 'no';
+}
+
+sub format_uplift_request_as_markdown {
+  my ($question_answers_mapping) = @_;
+
+  my $comment = '# Uplift Approval Request\n';
+
+  while (my ($question, $answer) = each %{$question_answers_mapping}) {
```

Comment:
Hmmm yes it would be better to have the comments always in the same order. Maybe I could change the storage from a mapping to a list of `(question, answer)` pairs on the Phabricator side...

For now I think we can just ship this as-is and I can fix it when it becomes a problem. :)